tla/MCccfraft.cfg

```diff
@@ -67,6 +67,7 @@ INVARIANTS
     MonoLogInv
     LogConfigurationConsistentInv
     NoLeaderInTermZeroInv
+    MatchIndexLowerBoundNextIndexInv
```

Comment:
Aside: if there any easy way to deduplicate these cfg files?

Reply:
Unfortunately not at the moment.